node/hyperbahn/service_proxy.js

```diff
@@ -42,6 +43,10 @@ function ServiceDispatchHandler(options) {
 
     self.egressNodes.on('membershipChanged', onMembershipChanged);
 
+    self.permissionsCache = new PermissionsCache({
```

Comment:
Let's not place this here. Let's just take `self.permissionsCache = options.permissionsCache;`

We can then instantiate it in autobahn/clients/index.js